src/main/java/org/junit/runner/Request.java

```diff
@@ -131,8 +131,11 @@ public Request filterWith(Filter filter) {
     }
 
     /**
-     * Returns a Request that only runs contains tests whose {@link Description}
-     * equals <code>desiredDescription</code>
+     * Returns a Request that only runs tests whose {@link Description}
+     * matches <code>desiredDescription</code>
+     *
+     * Returns an empty Request if <code>desiredDescription</code> is not a single test and filters all but the single
```

Comment:
Use `<p>` and `@code`:

    * <p>Returns an empty {@code Request} if {@code desiredDescription} is not...